ui/src/modules/Circles/Comparation/Item/helpers.ts

```diff
@@ -32,6 +33,14 @@ export const pathCircleById = (id: string) => {
 
 export const isDefaultCircle = (name: string) => name === DEFAULT_CIRCLE;
 
+export const isDeploying = (status: string) =>
```

Comment:
I'm just wondering, this way sounds more concise for me.

What do u think?

```suggestion
export const isDeploying = (status: CIRCLE_STATUS) =>
```